package.json

```diff
@@ -78,9 +79,11 @@
     "babel-core": "^6.17.0",
     "babel-eslint": "^7.1.1",
     "babel-loader": "^6.2.10",
+    "babel-macros": "^1.1.1",
```

Comment:
Note: `babel-macros` has had a name change to improve compatibility with babel v7. It's now `babel-plugin-macros`. That's the only breaking change. It's currently on `2.0.0`.